.github/workflows/build-deploy-all.yml

```diff
@@ -48,7 +42,7 @@ jobs:
       # Install dependencies
       - name: Install dependencies
         run: |
-          pipx install twine bump2version
+          pipx install twine
```

Comment:
It looks like the stop calling `bump2version` is still part of this workflow, depending on the removed `version_type` input

Reply:
We will manually submit a PR for a version bump going forward. Doing the version bump within the GHA means we have to push the update back to the repo. Previously this was done with a direct push to main, which I don't like ([example](https://github.com/jozu-ai/pykitops/commit/5c3388de32eb2bc565dcb26194f8f57219bd2e84)). We could do a PR instead but for now would just rather do it upfront. I have thought about this job upgraded to the next version and submitting a PR, but wanted to quickly remove the push to main. If it becomes to annoying, I can always adjust it later on.

Reply:
What I mean is that there's still a step (not part of this PR so I can't comment on it) that depends on bump2version (lines 53-58 in the current commit). The input has been removed, so this will probably fail.

```yaml
      # Bump version based on input
      - name: Bump version
        run: |
          bump2version ${{ github.event.inputs.version_type }}
        env:
          GITHUB_TOKEN: ${{ secrets.GH_TOKEN }}
```